lib/Server.js

```diff
@@ -390,13 +390,25 @@ class Server {
 
     if (Array.isArray(contentBase)) {
       contentBase.forEach((item) => {
-        this.app.use(contentBasePublicPath, serveIndex(item));
+        this.app.use(contentBasePublicPath, (req, res, next) => {
+          if (req.method !== 'GET') {
```

Comment:
Not sure it is good solution, developers can want to handle `GET` in own middleware too

Reply:
AFAIK, `serve-index` let `GET` request that it can't handle fall-back to the next middleware, it's just non-GET request that are blindly replied as `405`.

This is basically a revert of the change from `app.get` to `app.use` done in #2150 while still keeping the `contentBasePublicPath` feature.

Reply:
```suggestion
          if (req.method !== 'GET' && req.method !== 'HEAD') {
```